framework/src/limiters/Limiter.C

```diff
@@ -61,11 +61,24 @@ limiterType(const InterpMethod interp_method)
   switch (interp_method)
   {
     case InterpMethod::Average:
+    case InterpMethod::SkewCorrectedAverage:
```

Comment:
I think we should only have interpolation type. VanLeer MinMod and the others already indicate limiters. I might touch these when I dive in to the nonorthogonal correctors. We might need gradient limiters too.

Reply:
You mean there should not be a `SkewCorrected` in here? I agree. But if I remove this at the current time, we get errors since `skewness-corrected` in a user facing enumeration option in our NSFV interpolation methods

Reply:
There is a ton of redundancy between `InterpMethod` and `LimiterType`. I think this bifurcation appeared at the time we developed CNSFV because at that time it was not a priority to make sure that INSFV could handle all the same interpolation methods as CNSFV. But yea my test here for example shows just how redundant they are...
```c++
TEST(LimitersTest, limiterType)
{
  EXPECT_TRUE(limiterType(InterpMethod::Average) == LimiterType::CentralDifference);
  EXPECT_TRUE(limiterType(InterpMethod::SkewCorrectedAverage) == LimiterType::CentralDifference);
  EXPECT_TRUE(limiterType(InterpMethod::Upwind) == LimiterType::Upwind);
  EXPECT_TRUE(limiterType(InterpMethod::VanLeer) == LimiterType::VanLeer);
  EXPECT_TRUE(limiterType(InterpMethod::MinMod) == LimiterType::MinMod);
  EXPECT_TRUE(limiterType(InterpMethod::SOU) == LimiterType::SOU);
  EXPECT_TRUE(limiterType(InterpMethod::QUICK) == LimiterType::QUICK);
}
```